### PR TITLE
Juan backend

### PR DESCRIPTION
**Pull request by juan-maringka**

### CODE REVIEW

File: storage.py [hunk 1]
```diff
@@ -81,3 +216,5 @@ class Activities(Collection):
     def __init__(self):
         super().__init__("Activies")
```
Comment (birdyjellyfish) [inline storage.py:217]: Activities

Comment (birdyjellyfish) [inline storage.py:217]: super().__init__("Activities")

Comment (birdyjellyfish) [inline storage.py:217]: spell activities wrongly haha

**Review by birdyjellyfish — COMMENTED** [review]
got some problems hahaha

File: storage.py [hunk 2]
```diff
@@ -10,53 +11,134 @@ class Collection:
...
         super().__init__("Classes")
 
+    def add(self, name):
+        """"""
+        pass
+
+    def get_all(self, class_name):
+        """Returns all students in the corresponding class."""
+        # check if class exists
+        query = f"""
```
[the remark this answers is not in the document]
Comment (juan-maringka) [reply storage.py]: how do i do this

[the remark this answers is not in the document]
Comment (birdyjellyfish) [reply storage.py]: select class_name from classes table using return_row function. Then check if row object returned is None or not. Can create helper function for this too. 
image

**Pull request closed without merging**